spec/spec_helper_acceptance.rb

```diff
@@ -53,12 +57,12 @@
             on(host, 'apt-get install -y zlib1g-dev patch g++')
           end
 
-          gem_command = if host['platform'] =~ /^windows/
+          gem_command = if is_windows?(host)
                           'cmd.exe /c cmd.exe /c "C:\Program Files\Puppet Labs\puppet\sys\ruby\bin\gem.bat"'
                         else
                           host.file_exist?("#{host['privatebindir']}/gem") ? "#{host['privatebindir']}/gem" : "#{host['puppetbindir']}/gem"
```

Comment:
Unrelated to this change, but just in case you hit it, https://github.com/puppetlabs/puppetlabs-vsphere/commit/ffcff50ce98fbe42e6e17f44aec9091c3a4b24a8. Depends on what Puppet versions are being tested.